pom.xml

```diff
@@ -63,6 +63,30 @@
             <organization>Red Hat</organization>
             <organizationUrl>https://www.redhat.com</organizationUrl>
         </developer>
+        <developer>
+			<name>Stanislav Knot</name>
+			<email>knot@cngroup.dk</email>
+			<organization>CN Group</organization>
+			<organizationUrl>https://www.cngroup.dk/</organizationUrl>
+		</developer>
+		<developer>
+			<name>Paul Mellor</name>
+			<email>pmellor@redhat.com</email>
+			<organization>Red Hat</organization>
+			<organizationUrl>https://www.redhat.com</organizationUrl>
+		</developer>
+		<developer>
+			<name>Lukáš Král</name>
+			<email>l.kral@outlook.com</email>
+			<organization>Red Hat</organization>
+			<organizationUrl>https://www.redhat.com</organizationUrl>
+		</developer>
+		<developer>
+			<name>Maroš Orsák</name>
+			<email>maros.orsak159@gmail.com</email>
+			<organization>Red Hat</organization>
+			<organizationUrl>https://www.redhat.com</organizationUrl>
+		</developer>
```

Comment:
should it be aligned the same way as the other maintainers?

Reply:
I double checked in my IDE, it's aligned :-( Don't know why it's this way here. Let me try to re-copy and paste again.

Reply:
Should be better now :-)